tests/test_config.py

```diff
@@ -80,6 +80,11 @@ def test_dont_load_env():
     del environ["SANIC_TEST_ANSWER"]
 
 
+def test_load_env_deprecation():
+    with pytest.warns(DeprecationWarning, match=r"21\.12"):
+        _ = Sanic(name=__name__, load_env=False)
```

Comment:
I would `pytest.mark.parameterize` this to test with `False` and also a string: `FOO_`

Reply:
:heavy_check_mark: